documentation/latest/dev/guide.md

```diff
@@ -36,11 +44,14 @@ make push-images-to-kind
 kubectl kustomize hack/manifests/testing | kubectl apply -f -
 kubectl wait --timeout=300s --for=condition=Ready pods --all -n default
 ```
-Verify if all the resources has been correctly create and if they are running, execute 
+
+Verify if all the resources has been correctly create and if they are running,
+execute:
```

Comment:
let include also the short-cut for deploying a fix to the workflow-server/notification-server by running

```
kubectl kustomize hack/manifests/testing | kubectl delete -f -

make build-images
make tag-images
make push-images-to-kind
kubectl kustomize hack/manifests/testing | kubectl apply -f -
```
and as a fallback, to delete and create new kind cluster.

Reply:
see related [commit](https://github.com/parodos-dev/parodos-dev.github.io/pull/11/commits/a4ddac50a03bf48316b993889da1b9415804519b)